protocol/Cargo.toml

```diff
@@ -48,6 +48,7 @@ base64 = { version = "0.22", default-features = false }
 serde = { version = "1", default-features = false }
 serde_json = "1"
 postcard = { version = "1", default-features = false }
+prost = { version = "0.12", default-features = false }
```

Comment:
in opposite, please remove this dependency from platform/Cargo.toml, and migrate all imports `use prost::*` to use `sdk::cosmos_sdk_proto::prost::*`

Reply:
As mentioned before, if I could, I would. The `Message` derivation macro expects `prost` as a dependency and not as an import.